src/app/server/Mdns.h

```diff
@@ -32,6 +33,8 @@ CHIP_ERROR AdvertiseCommisioning();
 /// (Re-)starts the minmdns server
 void StartServer();
 
+CHIP_ERROR GenerateRotatingDeviceId(char rotatingDeviceIdHexBuffer[], size_t & rotatingDeviceIdHexBufferSize);
```

Comment:
Is Rotating device ID part of MDNS?

Feels slightly misplaced here, but unsure what the better place would be.